README.md

```diff
@@ -16,4 +16,8 @@ Basic statistics and data analysis with Java
 - <a href="#">UseContinuousSampleExample</a> Show cases how to create a ```NumericSample```
 - <a href="#">LoadCsvDataSetWithTreeMap</a>: How to load  CSV data and put a column in a ```NumericSample```
 - <a href="#">MeanBootstrapExample</a>: Vanilla Bootstrap for the mean
+- <a href="#">PlotALine</a>: Plot a line
```

Comment:
Can you be a bit more verbose here? E.g. ```Shows how to plot a line blah blah blah....```

Reply:
ok!